guara/transaction.py

```diff
@@ -3,102 +3,62 @@
 # terms of the MIT license.
 # Visit: https://github.com/douglasdcm/guara
 
```

Comment:
Do not change this file, please. It is already tested and working. If you face some issue, open a new PR or a ticket, please